android/HardwareUtils_Android.cpp

```diff
@@ -56,8 +58,9 @@ class FbListener: public ::firebase::messaging::Listener
 
     virtual void OnMessage(const ::firebase::messaging::Message &message)
     {
-        Q_UNUSED(message)
         qDebug() << "Received FCM message";
+        NotificationService *ns = new NotificationService();
```

Comment:
L'objet `ns` est détruit ou et par qui? Si il faut le faire nous meme, ce n'est pas le cas et ca leak.

Reply:
Singleton is used now.